app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/triage/capcitors_personal_representatives_not_handled.scala.html

```diff
@@ -0,0 +1,107 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.config.ViewConfig
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.actions.RequestWithSessionData
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.returns.triage
+
+@this(
+        mainTemplate: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.main_template,
+        pageHeading: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.page_heading,
+        backLinkComponent: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.back_link
+)
+
+@()(implicit request:RequestWithSessionData[_], messages:Messages, appConfig: ViewConfig)
+
+@key = @{"capacitorsPersonalRepresentativesNotHandled"}
+@title = @{messages(s"$key.title")}
+
+@mainTemplate(title = title, userType = request.userType) {
+
+ @backLinkComponent(triage.routes.InitialTriageQuestionsController.whoIsIndividualRepresenting())
+
+ @pageHeading(title)
+
+ <p>
+  @messages(s"$key.cannotUse")
+ </p>
+
+ <ul class="list-bullet">
+  <li>@messages(s"$key.cannotUse.bullet1")</li>
+  <li>@messages(s"$key.cannotUse.bullet2")</li>
+ </ul>
+
+ <p>
+  @messages(s"$key.youNeedTo")
+ </p>
+
+ <ul class="list-bullet">
+  <li>@messages(s"$key.youNeedTo.bullet1")</li>
+  <li>@messages(s"$key.youNeedTo.bullet2")</li>
+ </ul>
+
+ <h2 class="heading-medium">
+  @messages(s"$key.phoneHeading")
+ </h2>
+
+ <p>
+  @messages(s"$key.phoneHmrc")<br>
+  <strong class="bold">
+   @appConfig.hmrcTelephone
+  </strong>
+ </p>
+
+ <p>
+  @messages(s"$key.phoneOutsideUk")<br>
+  <strong class="bold">
+   @appConfig.outsideUkPhone
+  </strong>
+ </p>
+
+ <p>
+  @messages(s"$key.phoneAgent")<br>
+  <strong class="bold">
+   @appConfig.agentDedicatedLine
+  </strong>
+ </p>
+
+ <p>
+  @messages(s"$key.openingTimes")
+ </p>
+
+ <p>
+  @messages(s"$key.openingTimes.monToFri")
+ </p>
+
+ <p>
+  @messages(s"$key.openingTimes.saturday")
+ </p>
+
+ <p>
+  @messages(s"$key.openingTimes.sunBankHols")
+ </p>
+
+  <p>
+   @Html(messages(s"$key.getHelp", appConfig.additionalNeedsUrl))
+  </p>
+
+  <p>
+   <a href="@{appConfig.callChargesUrl}">
+    @messages(s"$key.callChargesLink")
```

Comment:
could we not do this like above ?
@Html(messages(s"$key.callChargesLink", appConfig. callChargesUrl))

Reply:
done - I think the reason the two was different is because the first link is a link mid-sentence but the second link was not